Templates/DefaultProject/Template/export.bat

```diff
@@ -0,0 +1,47 @@
+@echo off
+REM --------------------------------------------------------------------------------------------------
+REM 
+REM Copyright (c) Contributors to the Open 3D Engine Project.
+REM For complete copyright and license terms please see the LICENSE at the root of this distribution.
+REM 
+REM SPDX-License-Identifier: Apache-2.0 OR MIT
+REM 
+REM --------------------------------------------------------------------------------------------------
+
+REM This packaging script simplifies the project export command by defaulting the required and important
+REM arguments to the values based on:
+REM     --project-path %O3DE_PROJECT_PATH%      <- The path to this project for exporting
+REM     --tools-build-path %TOOLS_BUILD_PATH%   <- The location of the tools/editor build to make sure all the tools necessary 
+REM                                                for packaging are built and available (if applicable)
+REM     -out %OUTPUT_PATH%                      <- The output location for the exported project
+REM
+REM Feel free to adjust any of the arguments as necessary. For more information about the project export command, type in the 
+REM following command from the engine root
+REM
+REM %O3DE_PATH%\scripts\o3de.bat export-project -es ExportScripts\export_source_built_project.py --script-help
+REM
+REM To see the default options the export-project command will use other than the above required override arguments, you can view/edit
+REM the parameters with the export-project-configure command
+REM
+REM %O3DE_PATH%\scripts\o3de.bat export-project-configure --help
+REM
+REM To view the settings for the current project:
+REM
+REM %O3DE_PATH%\scripts\o3de.bat export-project-configure -p %O3DE_PROJECT_PATH% --list
+REM
+REM Note: The location of the engine (O3DE_PATH) is hardcoded to the location of the engine that was used to generate 
+REM       this project. The engine path must reflect the path to the engine on the local machine.
+
+set O3DE_PATH=${EnginePath}
+set O3DE_PROJECT_PATH=%~dp0
+set OUTPUT_PATH=%O3DE_PROJECT_PATH%\ProjectPackages
+set TOOLS_BUILD_PATH=%O3DE_PROJECT_PATH%\build\windows
```

Comment:
Good using the tools that have already been built. For future, would you consider updating the export-script.py itself to default to this locations instead of it's current default? build/tools/

Reply:
there was a discussion about this earlier about maybe using the values in the preset. However, we should consolidate that logic to ProjectManager as well for consistency.

Reply:
Yeah, ProjectManager's "build" button and the project-export-script should pull from the same variable when deciding where the build goes (ex: build/windows). For ProjectManager, tools and launchers will go into build/windows, whereas the project-export-script will just want to build its tools under build/windows.